frontend/src/model/beta/releases/ReleaseForm.tsx

```diff
@@ -63,93 +67,112 @@ export default function ReleaseForm({
     </Typography>
   )
 
   return (
-    <Stack spacing={2}>
-      {!editable && (
-        <Stack sx={{ width: '100%' }} justifyContent='center'>
-          <Stack direction='row'>
-            <Typography fontWeight='bold'>Release name</Typography>
-            {!editable && (
-              <HelpPopover>
-                The release name is automatically generated using the model name and release semantic version
-              </HelpPopover>
-            )}
+    <>
+      {isReleasesLoading && <Loading />}
+      <Stack spacing={2}>
+        {!editable && (
+          <Stack sx={{ width: '100%' }} justifyContent='center'>
+            <Stack direction='row'>
+              <Typography fontWeight='bold'>Release name</Typography>
+              {!editable && (
+                <HelpPopover>
+                  The release name is automatically generated using the model name and release semantic version
+                </HelpPopover>
+              )}
+            </Stack>
+            <Typography>{`${model.name} - ${formData.semver}`}</Typography>
           </Stack>
-          <Typography>{`${model.name} - ${formData.semver}`}</Typography>
-        </Stack>
-      )}
-      <Stack>
-        <Typography fontWeight='bold'>
-          Semantic version {!editable && <span style={{ color: 'red' }}>*</span>}
-        </Typography>
-        {isReadOnly || isEdit ? (
-          <ReadOnlyAnswer value={formData.semver} />
-        ) : (
-          <TextField
-            required
-            size='small'
-            error={formData.semver !== '' && !isValidSemver(formData.semver)}
-            helperText={formData.semver !== '' && !isValidSemver(formData.semver) ? 'Must follow format #.#.#' : ''}
-            value={formData.semver}
-            onChange={handleSemverChange}
-          />
-        )}
-      </Stack>
-      <Stack>
-        {isReadOnly ? (
-          <>
-            {releaseNotesLabel}
-            <MarkdownDisplay>{formData.releaseNotes}</MarkdownDisplay>
-          </>
-        ) : (
-          <RichTextEditor
-            value={formData.releaseNotes}
-            onChange={onReleaseNotesChange}
-            aria-label='Release notes'
-            label={releaseNotesLabel}
-          />
         )}
-      </Stack>
-      <Stack>
-        {isReadOnly || isEdit ? (
-          <>
-            <Typography fontWeight='bold'>Minor Release</Typography>
-            <ReadOnlyAnswer value={formData.isMinorRelease ? 'Yes' : 'No'} />
-          </>
-        ) : (
-          <FormControl>
-            <FormControlLabel
-              control={<Checkbox size='small' checked={formData.isMinorRelease} onChange={handleMinorReleaseChange} />}
-              label='Minor release - No significant changes, does not require release re-approval'
+        <Stack>
+          {!editable && (
+            <>
+              <Typography fontWeight='bold'>Latest semantic version</Typography>
```

Comment:
Up for discussion but I'd vote for this simply saying "Latest version". We don't need to drill home that it's a semver at every opportunity imo :)